src/compas/geometry/_core/angles.py

```diff
@@ -353,6 +354,34 @@ def angles_points_xy(a, b, c, deg=False):
     return angles_vectors_xy(u, v, deg)
 
 
+def angle_planes(a, b, deg=False):
+    """Compute the smallest angle between the two normal vectors of two planes.
+
+    Parameters
+    ----------
+    a : tuple of a point and a vector. Point as a sequence of float(XYZ) coordinates. Vector as a sequence of float(XY(Z)) coordinates.
+        first plane
+    b : tuple of a point and a vector. Point as a sequence of float(XYZ) coordinates. Vector as a sequence of float(XY(Z)) coordinates.
+        second plane
+    deg : boolean
+        returns angles in degrees if True
```

Comment:
please add capital letter at the start and period at the end.
angles => angle
True => ``True``